src/lib/GUID.cpp

```diff
@@ -70,20 +122,72 @@ bool GUID::is_equal(GUID other)
 }
 
 // This is used for FileDataStores. The GUID is given as a string in the file
-void GUID::from_string(std::string str)
+void GUID::from_string(std::string const str)
 {
   ONE_DEBUG_MSG(("\n"));
```

Comment:
nit: you can remove this if you want, it looks like it originated from code coming from ye olde ancient times

Reply:
ok.